dev-libs/gjs/gjs-1.75.90.ebuild

```diff
@@ -0,0 +1,73 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit flag-o-matic gnome.org meson virtualx
+
+DESCRIPTION="Javascript bindings for GNOME"
+HOMEPAGE="https://wiki.gnome.org/Projects/Gjs https://gitlab.gnome.org/GNOME/gjs"
+
+LICENSE="MIT || ( MPL-1.1 LGPL-2+ GPL-2+ )"
+SLOT="0"
+IUSE="+cairo examples readline sysprof test"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~riscv ~x86"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	>=dev-libs/glib-2.66.0:2
+	dev-libs/libffi:=
+	>=dev-libs/gobject-introspection-1.66.1:=
```

Comment:
````diff
-if gi.version().version_compare('<1.66.1')
+if gi.version().version_compare('<1.71.0')
````

We should raise the required version.

Reply:
Ok, also changed 
-       >=dev-libs/gobject-introspection-1.66.1:=
+       >=dev-libs/gobject-introspection-1.71.1:=